openpuc_scrapers/scrapers/ny.py

```diff
@@ -8,11 +8,15 @@
 from selenium import webdriver
 from selenium.webdriver.common.by import By
 from pydantic import BaseModel, HttpUrl
-from typing import List
+from typing import Any, Dict, List, Tuple
 import time
 from datetime import date, datetime
 from bs4 import BeautifulSoup
 
+from openpuc_scrapers.models.case import GenericCase
+from openpuc_scrapers.models.filing import GenericFiling
+from openpuc_scrapers.models.generic_scraper import GenericScraper
+
 
 class NYPUCAttachmentData(BaseModel):
     name: str
```

Comment:
@cappuccinocosmico is this the filename?

Reply:
There's also `filename`

Reply:
See [here](https://docs.google.com/spreadsheets/d/1aoV7W-6JD8GRk5NsbW8y6hmtsvaLYV2pOV8P80f9UJc/edit?gid=1331009742#gid=1331009742) for all `AttachmentNY` fields.